### ISSUE TITLE
Deleting/Reverting merged PR 

### ISSUE THREAD

**Issue by Anirban166** [Anirban166/testComplexity#39]
I introduced new icons for my resources section all of which had new names, but some of them (3 to be precise) were just different by the current pngs in `/Images` by the initial letter case in name, i.e. for eg: xyz.png was Xyz.png. I thought this didn't pose as an issue since they were treated as new files and when linking them to their raw github download URLs and viewing them through a `build_site()` operation in my RStudio session, all of them worked fine. After this I pushed quite a few commits with some updates to readme/index and the addition of a vignette (`PELT_and_SegNeigh.Rmd`) which was the main goal of the PR. I merged the PR right away thinking it would be fine, but sadly it wasn't. I looked at the job logs, the RCMD checks() failed at the image case names, where it signified that the previously (before the PR) available xyz.pngs were clashing with Xyz.pngs. This didn't make any sense because there were no xyz.pngs (as I deleted them as the first commit in the PR) in `/Images` and only the Xyz.pngs were present. It appears previously deleted files with similar names (only difference being the case for the initial letter) were still being somehow retrievable in job logs (Travis + Github actions). This was pretty frustrating so I decided to delete the commits & PRs and start over again.

I removed my commits for today (including 1 to master via github/remote branch directly, and 1 PR from local branch) first locally by hard resetting the HEAD to the sha1 signature of yesterday's last commit (cdc1c8d7bcca0d9400457a6296afb94ffd9afd8c) and then subsequently forcing the changes here to the github branch. As expected this did delete the commits, but the commit history for today is still filled, and the unwanted PRs (#37  & #38, the later was a result of a revert PR, another blunder) are still in testComplexity/PRs.

After looking up some SO threads and trying different git operations (I was pretty sure revert would be the same, and rebase a bit different but wouldn't work), including a filter-branch (to remove the dangling references from the reflogs) and garbage collection approach, I realized it can't be deleted from the Github UI i.e. a merged PR cannot along with its commit references cannot be removed from github. Regarding the same, I gave a ticket to github-support, and I will wait for a day to see if they can do it (will make another PR tomorrow).

Note for today: If the PR wouldn't have been merged, it is possible to remove the very existence of it (wouldn't have the reference). But once a PR is merged, we can delete our commits no doubt, but the activity and the PR status will remain on github.

**Comment by Anirban166** [comment]
The PRs have been deleted by Github Support with reference to my ticket - thanks to @tmetz !
